lib/commons/forms.less

```diff
@@ -83,23 +83,20 @@ textarea,
     color: @disabled;
   }
 
-  &@{prefix}required {
-    &:after {
-      content: 'Required';
-      display: inline-block;
-      margin-left: @space-large;
-      font-style: italic;
-      font-weight: @weight-normal;
-    }
-  }
-
   &@{prefix}error {
```

Comment:
The form validation adds the `"dqpl-error"` class to the label and before, changed the font color of the pseudo `::after` "Required" text. To keep that same functionality in place, please change this block to:

```
&@{prefix}required {
  @{prefix}required-text {
    color: @error;
  }
}
```